cli/internal/cmd/miniup.go

```diff
@@ -142,16 +143,18 @@ func (m *miniUpCmd) prepareConfig(cmd *cobra.Command, fileHandler file.Handler,
 			return nil, errors.New("not overwriting existing config")
 		}
 	}
-
-	config := config.Default()
-	config.Name = constants.MiniConstellationUID
-	config.RemoveProviderAndAttestationExcept(cloudprovider.QEMU)
-	config.StateDiskSizeGB = 8
-
+	if featureset.CurrentEdition != featureset.EditionEnterprise { // TODO: or !featureset.CanFetchMeasurements? image is also needed..
+		cmd.PrintErrln("Generating a valid default config is not supported in the OSS build of the Constellation CLI. Consult the documentation for instructions on where to download the enterprise version.")
+		return nil, errors.New("cannot create a mini cluster without a config file in the OSS build")
+	}
+	config := config.MiniDefault()
 	// only release images (e.g. v2.7.0) use the production NVRAM
 	if !config.IsReleaseImage() {
 		config.Provider.QEMU.NVRAM = "testing"
 	}
+	if err := config.Validate(flags.force); err != nil {
+		return nil, fmt.Errorf("validating config: %w", err)
+	}
```

Comment:
Maybe move both of these checks into `config.MiniDefault()`?
We probably always want to conditionally check what NVRAM to use, and we also always want to make sure the config we just generated is valid (which should be the case if it was generated with an enterprise CLI).